coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc

```diff
@@ -23,3 +23,12 @@ $form['display']['show_thumbnail'] = array(
     '0' => t('No', array(), array('context' => 'test')),
   ),
 );
+
+$form['display']['status'] = [
+    '#title' => t('Status', array(), array('context' => 'test')),
```

Comment:
we should use 2 space indentation here as we do in the Drupal standard.

Reply:
Somehow I completely overlooked these coding standards issues on this file. Fixed now!